doc/nrf/protocols/matter/getting_started/transmission_power.rst

```diff
@@ -0,0 +1,154 @@
+      For example, if you want to build for the ``nrf52840dk_nrf52840`` build target with the default Thread TX power equal to 2 dBm, add ``-DCONFIG_OPENTHREAD_DEFAULT_TX_POWER=2``.
+
+      See `nRF Connect for VS Code extension pack <How to work with build configurations_>`_ documentation for more information.
+
+   .. group-tab:: Command line
+
+      To build a Matter sample with a custom Thread TX power from the command line, add the :kconfig:option:`CONFIG_OPENTHREAD_DEFAULT_TX_POWER` Kconfig option variable and the dBm value to the build command.
+      For example, if you want to build for the ``nrf52840dk_nrf52840`` build target with the default Thread TX power equal to 2 dBm, run the following command:
+
+      .. code-block:: console
+
+         west build -b nrf52840dk_nrf52840 -- -DCONFIG_OPENTHREAD_DEFAULT_TX_POWER=2
+
+Changing TX power for Bluetooth LE
+**********************************
+
+To change the radio TX power used by Zephyr's Bluetooth LE controller, set the :kconfig:option:`CONFIG_BT_CTLR_TX_PWR` Kconfig option to the desired value.
+However, you cannot set this config value directly, as it obtains the value from the selected ``CONFIG_BT_CTLR_TX_PWR_MINUS_<X>`` or ``CONFIG_BT_CTLR_TX_PWR_PLUS_<X>``, where *<X>* is replaced by the desired power value, in an irregular dBm range from ``-40`` to ``8`` dBm.
+For example, to set Bluetooth LE TX power to +5 dBM, set the :kconfig:option:`CONFIG_BT_CTLR_TX_PWR_PLUS_5` Kconfig option to ``y``.
+
+For the list of possible values for ``CONFIG_BT_CTLR_TX_PWR_MINUS`` and ``CONFIG_BT_CTLR_TX_PWR_PLUS`` Kconfig options, as well as their dependencies, see the :ref:`kconfig-search`.
+The only exception is the value of 0 dBm, which is set with the :kconfig:option:`CONFIG_BT_CTLR_TX_PWR_0` Kconfig option.
+
+For multicore boards, the configuration must be applied to both the application and the network core images in a consistent manner.
+You can do this by either editing the :file:`prj.conf` file or build the sample with an additional argument in the following way:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
+
+      To build a Matter sample with a custom Bluetooth LE TX power in the nRF Connect for VS Code IDE, add the desired ``CONFIG_BT_CTLR_TX_PWR`` Kconfig option for both the application and the network cores to the build configuration's :guilabel:`Extra CMake arguments` and rebuild the build configuration.
+      To build for the network core, make sure to add the ``childImageName_`` parameter between ``-D`` and the name of the Kconfig option.
+      The parameter name varies depending on the devices you are building for.
+      For example:
+
+      * If you want to build for Thread devices for the ``nrf5340dk_nrf5340_cpuapp`` build target with a Bluetooth LE TX power equal to 4 dBm, add ``-DCONFIG_BT_CTLR_TX_PWR_PLUS_4=y`` and ``-Dmultiprotocol_rpmsg_CONFIG_BT_CTLR_TX_PWR_PLUS_4=y`` as CMake arguments.
```

Comment:
I've checked that the maximum BLE tx power for nRF5340 is CONFIG_BT_CTLR_TX_PWR_PLUS_3, and it can be provided only for network core configs. so:

```suggestion
      * If you want to build for Thread devices for the ``nrf5340dk_nrf5340_cpuapp`` build target with a Bluetooth LE TX power equal to ``3`` dBm, add ``-Dmultiprotocol_rpmsg_CONFIG_BT_CTLR_TX_PWR_PLUS_3=y`` as CMake arguments.
```